### PR TITLE
EBP improvements

### PR DESCRIPTION
**Pull request by PenghaiZhang**
#1527 #1524 

##### Checklist

- [x] the [contributor license agreement][] is signed
- [x] commit message follows [commit guidelines][]
- [ ] tests are included
- [x] screenshots are included showing significant UI changes
- [ ] documentation is changed or added

##### Description of change

1. Make some CSS changes in order to display three Wizard controls (Shuffle list, Auto-complete taxonomy and Grouping control) in full width with New UI turned on. 

2. Make Contribution right-hand navigation bar stick to the top of the page in New UI.  

Screenshot from 2020-06-23 16-00-05

Screenshot from 2020-06-23 16-00-56

### CODE REVIEW

**Comment by PenghaiZhang** [comment]
Well quite a few tests failed. Tomorrow I will fix them.

File: Source/Plugins/Core/com.equella.core/resources/web/sass/legacy.scss [hunk 1]
```diff
@@ -848,6 +848,23 @@ a[title="move-down"]:before {
   width: 476px;
 }
 
+@media only screen and (max-width: 768px) {
+  .wizard-layout .indent0 .shuffle-box-inner select{
```
Comment (ChristianMurphy) [inline Source/Plugins/Core/com.equella.core/resources/web/sass/legacy.scss:852]: Are 4 layers of specificity needed here?
Is there a class on the `select` which could be used over a tag?

---

```suggestion
  .wizard-layout .indent0 .shuffle-box-inner select {
```

File: Source/Plugins/Core/com.equella.core/resources/web/sass/legacy.scss [hunk 2]
```diff
@@ -848,6 +848,23 @@ a[title="move-down"]:before {
...
+    width: 30%;
+  }
+}
+
+@media only screen and (min-width: 769px) and (max-width: 1200px) {
+  .wizard-layout .indent0 .shuffle-box-inner select{
```
Comment (ChristianMurphy) [inline Source/Plugins/Core/com.equella.core/resources/web/sass/legacy.scss:858]:
```suggestion
  .wizard-layout .indent0 .shuffle-box-inner select {
```

File: Source/Plugins/Core/com.equella.core/resources/web/sass/legacy.scss [hunk 3]
```diff
@@ -848,6 +848,23 @@ a[title="move-down"]:before {
...
+    width: 38%;
+  }
+}
+@media only screen and (min-width: 1201px) {
+  .wizard-layout .indent0 .shuffle-box-inner select{
```
Comment (ChristianMurphy) [inline Source/Plugins/Core/com.equella.core/resources/web/sass/legacy.scss:863]:
```suggestion
  .wizard-layout .indent0 .shuffle-box-inner select {
```

File: Source/Plugins/Core/com.equella.core/resources/web/sass/legacy.scss [hunk 4]
```diff
@@ -2654,6 +2671,10 @@ Contribution wizard styling
   width: 221px;
   top: 55px;
 }
+.wizard-layout .rightNav {
```
Comment (ChristianMurphy) [inline Source/Plugins/Core/com.equella.core/resources/web/sass/legacy.scss:2674]:
```suggestion

.wizard-layout .rightNav {
```

File: Source/Plugins/Core/com.equella.core/resources/web/sass/legacy.scss [hunk 5]
```diff
@@ -848,6 +848,23 @@ a[title="move-down"]:before {
...
+    width: 30%;
+  }
+}
+
+@media only screen and (min-width: 769px) and (max-width: 1200px) {
+  .wizard-layout .indent0 .shuffle-box-inner select{
+    width: 38%;
+  }
+}
+@media only screen and (min-width: 1201px) {
```
Comment (ChristianMurphy) [inline Source/Plugins/Core/com.equella.core/resources/web/sass/legacy.scss:862]:
```suggestion

@media only screen and (min-width: 1201px) {
```

File: Source/Plugins/Core/com.equella.core/resources/web/sass/legacy.scss (hunk 1 above)
Comment (PenghaiZhang) [reply Source/Plugins/Core/com.equella.core/resources/web/sass/legacy.scss:852]: > Are 4 layers of specificity needed here?
> Is there a class on the `select` which could be used over a tag?

As far as I know, the `select` does not have its own class. 
image

**Comment by PenghaiZhang** [comment]
Many tests failed because the Wizard right-hand navigation bar did not display properly in Moderation page.
So this PR also fixes this issue and make the bar stick to top of the page, too.

Screenshot from 2020-06-24 12-21-29

**Comment by PenghaiZhang** [comment]
AdvancedScriptControlTests failed so I restarted it to see if it would fail again.

**Pull request merged**